parts/chapters/10.fodt

```diff
@@ -19214,18 +19214,18 @@ Updated with AFR/TSA Rev-D comments and new keywords.</dc:description><meta:init
 <text:h text:style-name="P17490" text:outline-level="1"><text:bookmark-start text:name="__RefHeading___Toc1620_3043861001"/>SOLUTION SECTION<text:bookmark-end text:name="__RefHeading___Toc1620_3043861001"/></text:h>
     <text:p text:style-name="_40_TextBody"/>
     <text:h text:style-name="P17541" text:outline-level="2"><text:bookmark-start text:name="__RefHeading___Toc37628_3701168388"/>Introduction<text:bookmark-end text:name="__RefHeading___Toc37628_3701168388"/></text:h>
-    <text:p text:style-name="P12284">The SOLUTION section defines the initial conditions for the model, that is the conditions at time zero, before any flow occurs in the reservoir. As such, sufficient information from the previous sections is required for the simulator to calculate the initial saturation and pressure distributions through out the model, such that the fluids are in gravity/hydrostatic equilibrium. This means that if the simulator is run for a period of time without any production or injection then the there should be no saturation or pressure changes in the model. <text:s/>In fact this “test” of running the simulator for a few years without any production or injection should always be performed to ensure that the model is indeed in hydrostatic equilibrium.</text:p>
+    <text:p text:style-name="P12284">The SOLUTION section defines the initial conditions for the model, that is the conditions at time zero, before any flow occurs in the reservoir. <text:s/>As such, sufficient information from the previous sections is required for the simulator to calculate the initial saturation and pressure distributions through out the model, such that the fluids are in gravity/hydrostatic equilibrium. This means that if the simulator is run for a period of time without any production or injection then there should be no saturation or pressure changes in the model. <text:s/>In fact this “test” of running the simulator for a few years without any production or injection should always be performed to ensure that the model is indeed in hydrostatic equilibrium.</text:p>
```

Comment:
Is the additional space <text:s/>  before As intended?

Reply:
LO seemed to me to make the space after the full stop too small without the additional space. But happy with or without it :)